model/metrics/process-metrics.yaml

```diff
@@ -102,3 +103,11 @@ groups:
     unit: "{fault}"
     attributes:
       - ref: process.paging.fault_type
+
+  - id: metric.process.uptime
```

Comment:
We have a separate issue open for this metric: #648 

Would be fine by me if this ended up being covered by this PR. I don't have a good grasp on how semconv generally wants these types of metrics to be designed; I see arguments for these kinds of `uptime` metrics being not useful since the metrics should be reported with a start time anyway. (I might not be fully understanding this here, so hopefully I'm not misrepresenting). Given that, I'd like to make sure I'm understanding the intended usage of the metric. I see it's a `counter`; does that mean each reported point is essentially reporting `currentTime` - `processStartTime`?

Reply:
I can understand the argument that the uptime metric might not be too useful. OSs don't track process uptime specifically, so it would be calculated by `currentTime - processStartTime`. However this has been requested a few times in various issues, and there are existing users of process.uptime in ECS, so I think that shows there's value in having this.

I'm not sure if this should be a counter or gauge. I picked counter because it's monotonic, but it is also non-additive, so it could also be a gauge.  It was previously discussed [here](https://github.com/open-telemetry/opentelemetry-specification/pull/2824#discussion_r978765127), but I don't see a resolution to that discussion. An individual process's uptime is monotonic, but in aggregation, a group of processes' uptime could decrease (if processes are stopped and replaced by younger processes). I don't know if the metric would be more useful to track individual process uptime, or an aggregation. @reyang, since you were involved in that discussion, do you remember the outcome?

Reply:
Without knowing what can be the objections here, a metric that would report a specific's process uptime makes sense to me as a user.